src/main/java/uk/gov/di/ipv/cri/common/library/domain/personidentity/PersonIdentity.java

```diff
@@ -1,16 +1,21 @@
 package uk.gov.di.ipv.cri.common.library.domain.personidentity;
 
 import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
 
 import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.List;
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class PersonIdentity {
     private String firstName;
     private String middleNames;
     private String surname;
     private List<Address> addresses;
+    private List<SocialSecurityRecord> socialSecurityRecords;
```

Comment:
That matches the weblink of the vocab ( https://vocab.account.gov.uk/v1/classes/SocialSecurityRecordDetailsClass/ ) but not the JSON property ( https://github.com/govuk-one-login/ipv-stubs/blob/main/di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/models/JarUserInfo.java#L11-L12 )

I'm not sure if they're equivalent, or if one of them is not named properly.

Reply:
I think the property in orch needs the json link updated to https://vocab.account.gov.uk/v1/slots/socialSecurityRecord/ then these should align

Reply:
Looks correct in core it appears to just be this stub which is out of date